src/EnergyPlus/WindowManager.cc

```diff
@@ -642,27 +636,22 @@ namespace Window {
                 }
             } // End of loop over glass layers in the construction for front calculation
 
-            if (TotalIPhi > maxIncidentAngles) {
+            if (TotalIPhi > numPhis) {
                 ShowSevereError(state,
                                 format("WindowManage::InitGlassOpticalCalculations = {}, Invalid maximum value of common incident angles = {}.",
```

Comment:
This check looks like it was intended for `Window::OpticalDataModel::SpectralAndAngle` but does that method even care how many `Phis` there are? Looks like it just accesses tables directly now. I'm pretty sure this if block can be removed. It became irrelevant in #6749 See [comments](https://github.com/NREL/EnergyPlus/pull/6749/files#r307857849).